rpc/src/lib.rs

```diff
@@ -120,6 +120,40 @@ pub trait RmrkApi<
 		at: Option<BlockHash>,
 	) -> RpcResult<Vec<NftChild<CollectionId, NftId>>>;
 
+	#[method(name = "nftsOwnedBy")]
+	/// Get all of the NFTs of the provided account. Supports pagination by
+	/// specifying an optional `start` and `count`.
+	///
+	/// The `start` parameter defines the number of collections after which we
+	/// start reading the NFTs. The collections in which the user doesn't own
+	/// any NFTs are not counted.
+	///
+	/// The `count` parameter specifies the number of collections to read from.
+	fn nfts_owned_by(
+		&self,
+		account_id: AccountId,
+		start: Option<u32>,
```

Comment:
Are the collections sorted? So if i own NFTs in 3 collections with IDs 1, 5 and 7 and I define `start` with as 2 then I would only get NFTs I own in collection id 7?

Reply:
Yes exactly. The collections are sorted inside `nftsOwnedBy` and `propertiesOfNftsOwnedBy`:
https://github.com/rmrk-team/rmrk-substrate/pull/263/files#diff-0ec06ea58bd455f09ce6b3bb4c2c1c0d37bda51c1e1be2151c560c9c973959ecR539-R540

Reply:
That makes sense. Do you think we should call it `index` instead of `start`?

Reply:
Good idea. This will make it more obvious what `start` is.